data/venue.yml

```diff
@@ -0,0 +1,21 @@
+name: Museum of New Zealand, Te Papa Tongarewa
+nickname: Te Papa
+address: 55 Cable Street, Te Aro, Wellington 6011, New Zealand
+url: https://www.tepapa.govt.nz/
+location:
+  lat: -41.2905022
+  long: 174.7817372
+maps:
+  url: https://goo.gl/maps/ZSehkwLEvk52
```

Comment:
This actually links to the Te Papa store - a better URL is https://goo.gl/maps/LPGG4pc91892 - I might just merge this and fix that up myself though